cql/DashboardLibrary.cql

```diff
@@ -852,17 +984,33 @@ define Errors:
 
 // SUMMARY
 
+define Pregnant:
+  Exists(PregnancyDiagnoses) or
+  Exists(PregnancyObservations)
+
 // The Summary object represents the full Cervical Cancer Screening and Management Medical History to be displayed to the clinician. A robust UI
 // should be created in order to display this information in a Cervical Cancer Screening and Management Dashboard
 
 define CervicalCancerScreeningAndMedicalHistory:
 {
-  Age: PatientAge,
-  Conditions: PertinentConditionsSummary,
-  Observations: PertinentObservationsSummary,
-  Medications: PertinentMedicationsSummary,
-  Procedures: PertinentProceduresSummary,
-  DiagnosticReports: DiagnosticReportsSummary,
-  Encounters: PertinentEncountersSummary,
-  Immunizations: PertinentImmunizationsSummary
+  patientInfo: {
+    name: First(Patient.name.given) + First(Patient.name.family),
+    id: Patient.identifier.value,
+    isPregnant: Pregnant,
+    dateOfBirth: Patient.birthDate,
+    sexAtBirth: null,
```

Comment:
@dwwinters Why are there null values here when we know how these should be represented according to US Core?